### ISSUE TITLE
Blend Previews not working on Playnite 10.5

### ISSUE THREAD

**Issue by darklinkpower** [JosefNemec/Playnite#3158]
### Bug Description

Blend Previews not working on Playnite 10.5 on DetailsViewGameOverview.xaml. I've not checked in which other files this issue occurs.

### To Reproduce

1. Open DetailsViewGameOverview.xaml
2. Try to open the preview

Error:

```
Microsoft.VisualStudio.DesignTools.Designer.InstanceBuilders.InstanceBuilderException: Cannot create an instance of "DetailsViewGameOverview". ---> System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation. ---> System.NotImplementedException: The method or operation is not implemented. ...	Theme	C:\Games\Playnite\Themes\Desktop\8b15c46a-90c2-4fe5-9ebb-1ab25ba7fcb1\Views\DetailsViewGameOverview.xaml	1		Microsoft.VisualStudio.DesignTools.Designer.InstanceBuilders.InstanceBuilderException: Cannot create an instance of "DetailsViewGameOverview". ---> System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation. ---> System.NotImplementedException: The method or operation is not implemented.
   at Playnite.Database.InMemoryGameDatabase.GetGameMatchesFilter(Game game, FilterSettings filterSettings)
   at Playnite.BaseCollectionView.Filter(Object item)
   at System.Windows.Data.ListCollectionView.PrepareLocalArray()
   at System.Windows.Data.ListCollectionView.RefreshOverride()
   at System.Windows.Data.CollectionView.RefreshInternal()
   at System.Windows.Data.CollectionView.Refresh()
   at System.Windows.Data.CollectionView.EndDefer()
   at System.Windows.Data.CollectionView.DeferHelper.Dispose()
   at Playnite.DesktopApp.DesktopCollectionView..ctor(IGameDatabaseMain database, PlayniteSettings settings, ExtensionFactory extensions)
   at Playnite.DesktopApp.ViewModels.DesignMainViewModel..ctor()
   at Playnite.DesktopApp.ViewModels.DesignMainViewModel.get_DesignIntance()
   at Playnite.DesktopApp.Controls.Views.GameOverview..ctor(DesktopView viewType, DesktopAppViewModel mainModel)
   at Playnite.DesktopApp.Controls.Views.DetailsViewGameOverview..ctor()
   --- End of inner exception stack trace ---
   at System.RuntimeTypeHandle.CreateInstance(RuntimeType type, Boolean publicOnly, Boolean noCheck, Boolean& canBeCached, RuntimeMethodHandleInternal& ctor, Boolean& bNeedSecurityCheck)
   at System.RuntimeType.CreateInstanceSlow(Boolean publicOnly, Boolean skipCheckThis, Boolean fillCache, StackCrawlMark& stackMark)
   at System.RuntimeType.CreateInstanceDefaultCtor(Boolean publicOnly, Boolean skipCheckThis, Boolean fillCache, StackCrawlMark& stackMark)
   at System.Activator.CreateInstance(Type type, Boolean nonPublic)
   at Microsoft.VisualStudio.DesignTools.Designer.InstanceBuilders.InstanceBuilderOperations.InstantiateType(Type type, Boolean supportInternal)
   --- End of inner exception stack trace ---
   at Microsoft.VisualStudio.DesignTools.Designer.InstanceBuilders.InstanceBuilderOperations.InstantiateType(Type type, Boolean supportInternal)
   at Microsoft.VisualStudio.DesignTools.Designer.Views.ViewObjects.ViewContentProvider.InstantiateTargetType(IAssembly projectAssembly, ILocalInstanceBuilderContext context, Type type)
   at Microsoft.VisualStudio.DesignTools.Designer.Views.ViewObjects.ViewContentProvider.InstantiateTargetType[T](IAssembly projectAssembly, ILocalInstanceBuilderContext context, Type type)
   at Microsoft.VisualStudio.DesignTools.WpfDesigner.Views.ViewObjects.TemplateContentProvider.ProvideContent(ContentProviderParameters contentProviderContext)
   at Microsoft.VisualStudio.DesignTools.WpfDesigner.Views.ViewObjects.WpfViewContentProvider.ProvideContent(ContentProviderParameters contentProviderContext)
```

### Diagnostics ID

-

### Screenshots

![image](https://user-images.githubusercontent.com/1389286/201256733-45bd2d33-38dc-4209-b0c0-f80ba3757596.png)

**Comment by JosefNemec** [comment]
I'm not sure how this is possible, can you try reinstalling Playnite to see if it fixes the issue?

**Comment by darklinkpower** [comment]
I just checked and found out:
- On my current install, the issue only happens on Blend 2017. Blend 2019 works fine. Weird thing is that I think I remember the issue happening on both versions before (In fact, I'm 99% sure the screenshot is from v2019 as that's the one I've pinned in my taskbar)
- On a completely fresh build and new theme created toolbox, it works fine on both Blend 2017 and 2018
- Previews work fine now on both versions after extracting and replacing fresh files into my current Playnite install

**Comment by darklinkpower** [comment]
Disregard. I forgot about it and extracted Playnite v10.3 files instead of latest version. The issue appears again after updating to the latest 10.6 version

**Comment by JosefNemec** [comment]
This could in theory only happen if there's a mismatch between Playnite's exe files and Playnite.dll file. But I don't have any idea how that could have happened.

**Comment by darklinkpower** [comment]
> This could in theory only happen if there's a mismatch between Playnite's exe files and Playnite.dll file. But I don't have any idea how that could have happened.

That was a mistake of mine. In the following comment to that one I state that the issue happens on fresh v10.6 too, on both Blend versions.

**Comment by JosefNemec** [comment]
Found the issue.

**Comment by JosefNemec** [comment]
Fixed in 10.7

**Comment by JosefNemec** [comment]
Fixed in 10.7
